scipy/io/arff/tests/data/test9.arff

```diff
@@ -0,0 +1,8 @@
+@RELATION test9
```

Comment:
Added as a separate file rather than to `test7.arff`, to avoid adding another field to every row in that file.